STYLE-GUIDE.md

```diff
@@ -0,0 +1,53 @@
+# Style Guide
+
+This document defines some general styles we adhere to in the docs.
+
+## Words are important
```

Comment:
consider throwing in a link to G's "word list" in their dev guide ... it's much more thorough and offers rationale, alternates, exceptions, etc